### DISCUSSION TITLE
Editing Address

### DISCUSSION THREAD

**Discussion by puebloaa** [code4recovery/12-step-meeting-list#1009] [Q&A] [answered]
I am having a problem with our website "www.puebloaa.org".  The address for our club is wrong.  Whenever I edit the address to the correct address, it reverts back to the wrong address and shows this address for all meetings at the club.  Is there any way that I can edit this information?

**Comment by tim-rohrer** [comment, marked as answer]
My first guess is that something isn't happening with the geocoding, but I have some questions to ask to make sure I understand the situation. If you can post some screen caps, that might help.

So you type in the new address, and it reverts right away? Or does it geocode and show on the map. Afterwards, you click `Publish` or `Update`, and then it reverts?

What is the address you're trying to enter?

**Reply by puebloaa** [reply]
I found the answer, the club was listed twice, once with the right address and once with the wrong address.
